server/src/main/java/org/opensearch/node/NodeService.java

```diff
@@ -140,6 +143,7 @@ public class NodeService implements Closeable {
         this.fileCache = fileCache;
         this.taskCancellationMonitoringService = taskCancellationMonitoringService;
         this.resourceUsageCollectorService = resourceUsageCollectorService;
+        this.remoteClusterStateService = remoteClusterStateService;
```

Comment:
why do you need this?

Reply:
Mistakenly pushed it, I was using it when I was trying to publish the stats as a separate section. Not needed now.